### PR TITLE
Feat/add firebase

### PR DESCRIPTION
**Pull request by Anexon**

### CODE REVIEW

File: src/app/shared/services/community.service.ts [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
-  constructor(private httpClient: HttpClient) {}
+  constructor( public firestore: AngularFirestore) {
+
+  }
 
   get communities(): Observable<Communities> {
-    return this.httpClient
-      .get<Communities>(this.JSON_COMMUNITIES)
-      .pipe(map(communities => this.normalizeCommunities(communities)));
+    return this.firestore.collection(environment.firestore.COMMUNITIES_PATH).valueChanges().pipe(
```
Comment (tonivj5) [inline src/app/shared/services/community.service.ts:19]: @Anexon, is it reactive? When a new inserting on firebase is performed, it emits a new value?

Comment (Anexon) [reply src/app/shared/services/community.service.ts:19]: @tonivj5 exactly

Comment (Anexon) [reply src/app/shared/services/community.service.ts:19]: @tonivj5 if you need the credential files of firebase and google storage let me know and I can help you getting them from the AngularCommunity, account.
As well as adding your email as admin to log in successfully

Comment (tonivj5) [reply src/app/shared/services/community.service.ts:19]: Hey, yeah, how can I do it?

Comment (Anexon) [reply src/app/shared/services/community.service.ts:19]: It is a little bit tricky writing it down :D, would it be possible for you to do a quick hangout at some time today?

Comment (tonivj5) [reply src/app/shared/services/community.service.ts:19]: Sure, I can :+1:

**Pull request closed without merging**